src/Tests/Microsoft.Diagnostics.Monitoring.OpenApiGen/Program.cs

```diff
@@ -38,17 +38,18 @@ public static void Main(string[] args)
             // Create directory if it does not exist
             Directory.CreateDirectory(Path.GetDirectoryName(outputPath));
 
+            HostBuilderSettings settings = HostBuilderSettings.CreateMonitor(
+                urls: null,
+                metricUrls: null,
+                metrics: false,
```

Comment:
Just double checking that this change from `true` to `false` is intentional

Reply:
The setting here doesn't impact the functionality of this executable (to generate the OpenAPI document).